payjoin/src/send/v2/error.rs

```diff
@@ -2,6 +2,7 @@ use core::fmt;
 
 use crate::uri::url_ext::ParseReceiverPubkeyParamError;
 
+pub type ImplementationError = Box<dyn std::error::Error + Send + Sync>;
```

Comment:
Since this duplicates receiver::ImplementationError would be nice to de-duplicate in lib.rs or a core/error.rs module in a follow up.

Reply:
If we can do this in a follow up i would prefer that